docs/en/changelog.md

```diff
@@ -1,5 +1,127 @@
 ## Changelog
 
+### v2.24.0 (26/4/2022)
+
+#### Highlights
+
+- Support [Simple Copy-Paste is a Strong Data Augmentation Method for Instance Segmentation](https://arxiv.org/abs/2012.07177)
+- Support automatically scaling LR according to GPU number and samples per GPU
+- Support Class Aware Sampler
+
+#### New Features
+
+- Support [Simple Copy-Paste is a Strong Data Augmentation Method for Instance Segmentation](https://arxiv.org/abs/2012.07177), see [example configs](configs/simplecopypaste/mask_rcnn_r50_fpn_syncbn-all_rpn-2conv_ssj_scp_32x2_270k_coco.py). (#7501)
+- Support Class Aware Sampler, users can set
+
+  ```python
+  data=dict(train_dataloader=dict(class_aware_sampler=dict(num_sample_class=1))))
+  ```
+
+  in the config to use `ClassAwareSampler`. Examples can be found in [the configs of OpenImages Dataset](https://github.com/open-mmlab/mmdetection/tree/master/configs/openimages/faster_rcnn_r50_fpn_32x2_cas_1x_openimages.py)  (#7436)
+
+- Support automatically scaling LR according to GPU number and samples per GPU. In each config, there is a corresponding config of auto-scaling LR as below,
+
+   ```python
+   auto_scale_lr = dict(enable=True, base_batch_size=N)
+   ```
+
+  where `N` is the batch size used for the current learning rate in the config (also equals to `samples_per_gpu` * gpu number to train this config).
+  By default, we set `enable=False` so that the original usages will not be affected. Users can set `enable=True` in each config to enable this feature and should check the correctness of `base_batch_size` in customized configs. (#7482)
```

Comment:
Users can set `enable=True` in each config or add `--auto-scale-lr` after the command line to enable this feature and should check the correctness of `base_batch_size` in customized configs.